### PR TITLE
Feat: DS core stories

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- updates DS core components story format

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/design-system-core/src/components/CircularPlaceholder/circularPlaceholder.stories.tsx
Comment (didd) [inline ui/design-system-core/src/components/CircularPlaceholder/circularPlaceholder.stories.tsx:1]: It seems you named this file starting with a small cap :)

File: ui/design-system-core/src/components/ContentBlock/contentBlock.stories.tsx
Comment (didd) [inline ui/design-system-core/src/components/ContentBlock/contentBlock.stories.tsx:1]: File named starting with small caps

File: ui/design-system-core/src/components/SeventyFivePercentSpinner/index.tsx [hunk 1]
```diff
@@ -28,7 +25,7 @@ const SeventyFivePercentSpinner: React.FC<ISpinnerProps> = props => {
           `inline-block ${
             spinnerSizesMap[size]
           } animate-spin rounded-full border-4 border-solid ${getColorClasses(
-            color,
+            color as Color,
```
Comment (didd) [inline ui/design-system-core/src/components/SeventyFivePercentSpinner/index.tsx]: Is this really needed ?

File: ui/design-system-core/src/components/Spinner/index.tsx [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 import { tw } from '@twind/core';
 
-export interface ISpinnerProps {
+import { Color } from '../types/common.types';
+
+export type SpinnerProps = {
   size?: 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
-  color?: string;
-}
+  color?: string | Color;
```
Comment (didd) [inline ui/design-system-core/src/components/Spinner/index.tsx]: Using string will lead a dev to a use a color that's not defined or known by tailwind or twind. That's why we are explicitly defining colors we are using on Colors type.

File: ui/design-system-core/src/components/Spinner/index.tsx [hunk 3]
```diff
@@ -34,11 +37,11 @@ const Spinner: React.FC<ISpinnerProps> = props => {
         />
         <path
           d="M93.9676 39.0409C96.393 38.4038 97.8624 35.9116 97.0079 33.5539C95.2932 28.8227 92.871 24.3692 89.8167 20.348C85.8452 15.1192 80.8826 10.7238 75.2124 7.41289C69.5422 4.10194 63.2754 1.94025 56.7698 1.05124C51.7666 0.367541 46.6976 0.446843 41.7345 1.27873C39.2613 1.69328 37.813 4.19778 38.4501 6.62326C39.0873 9.04874 41.5694 10.4717 44.0505 10.1071C47.8511 9.54855 51.7191 9.52689 55.5402 10.0491C60.8642 10.7766 65.9928 12.5457 70.6331 15.2552C75.2735 17.9648 79.3347 21.5619 82.5849 25.841C84.9175 28.9121 86.7997 32.2913 88.1811 35.8758C89.083 38.2158 91.5421 39.6781 93.9676 39.0409Z"
-          fill={color}
+          fill={color as string}
```
Comment (didd) [inline ui/design-system-core/src/components/Spinner/index.tsx]: If we change how we define the type of the props, the `as string` can be avoided.

**Comment by didd** [comment]
Other than the comments above, all else looks fine to me

File: ui/design-system-core/src/components/CircularPlaceholder/circularPlaceholder.stories.tsx
Comment (josenriagu) [reply ui/design-system-core/src/components/CircularPlaceholder/circularPlaceholder.stories.tsx:1]: yeap, to fix the warning about kebab-case

File: ui/design-system-core/src/components/ContentBlock/contentBlock.stories.tsx
Comment (josenriagu) [reply ui/design-system-core/src/components/ContentBlock/contentBlock.stories.tsx:1]: same kebab case warning about file name

**Review by didd — APPROVED** [review]

**Pull request merged**